--- ImageMagick-6.9.13-9/PerlMagick/Makefile.PL.in.orig	2024-04-19 20:13:48.000000000 +0200
+++ ImageMagick-6.9.13-9/PerlMagick/Makefile.PL.in	2024-04-24 20:57:29.699101806 +0200
@@ -223,9 +223,6 @@ WriteMakefile
    # Linker flags for building a dynamically loadable module
    'LDDLFLAGS' => $LDDLFLAGS_magick,
 
-   # Install PerlMagick into ImageMagick prefix
-   'INSTALL_BASE' => '@PREFIX_DIR@',
-
    # Install PerlMagick binary into ImageMagick bin directory
    'INSTALLBIN'	=> '@BIN_DIR@',
 
