--- ImageMagick-5.4.9/magick/utility.c.orig	Sun Sep  8 01:53:34 2002
+++ ImageMagick-5.4.9/magick/utility.c	Mon Jul  7 20:20:54 2003
@@ -2678,6 +2678,39 @@
   geometry->width=image->columns;
   geometry->height=image->rows;
 }
+
+ MagickExport void TemporaryFilename(char *path)
+ {
+   static char
+     *mSafeTmpdir = NULL;
+
+   char
+     *name;
+
+   if (!mSafeTmpdir)
+   {
+     do
+     {
+       TemporaryFilenameHelper(path);
+       if (mkdir(path, S_IRWXU) == 0)
+       {
+         mSafeTmpdir = strdup(path);
+         break;
+       }
+     } while (errno == EEXIST);
+   }
+
+   /* FIXME: Need to address VMS and older MacOS */
+   name = tempnam(mSafeTmpdir, (char *) NULL);
+   if (!name || !mSafeTmpdir)
+   {
+     path[0] = '\0';
+     return;
+   }
+   (void) strncpy(path,name,MaxTextExtent-1);
+   free(name);
+ }
+
 
 /*
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
@@ -3215,7 +3248,7 @@
 %      file name is returned in this array.
 %
 */
-MagickExport void TemporaryFilename(char *filename)
+static void TemporaryFilename(char *filename)
 {
   assert(filename != (char *) NULL);
   (void) strcpy(filename,"magic");
