--- ImageMagick-6.2.1/coders/Makefile.am.orig	2005-03-27 23:57:03.062415000 +0200
+++ ImageMagick-6.2.1/coders/Makefile.am	2005-03-28 00:38:39.201944952 +0200
@@ -349,7 +349,7 @@
 coders_dot_la_SOURCES      = coders/dot.c
 coders_dot_la_CPPFLAGS     = $(MODULE_EXTRA_CPPFLAGS)
 coders_dot_la_LDFLAGS      = $(MODULECOMMONFLAGS)
-coders_dot_la_LIBADD       = $(LIBMAGICK) 
+coders_dot_la_LIBADD       = $(LIBMAGICK) $(LIB_DOT)
 
 # EMF coder module
 coders_emf_la_SOURCES      = coders/emf.c
