--- ./ltdl/Makefile.am.org	2005-09-28 17:06:38.000000000 +0000
+++ ./ltdl/Makefile.am	2005-09-28 17:07:54.000000000 +0000
@@ -28,12 +28,8 @@
 AUTOMAKE_OPTIONS = no-dependencies foreign
 
 if WITH_LTDL
-  LIBLTDL = ltdl/libltdl.la
-if INSTALL_LTDL
-  LTDL_INSTALLHEADERS = $(LTDL_HDRS)
-else
+  LIBLTDL=-lltdl
   LTDL_NOINST_HDRS = $(LTDL_HDRS)
-endif # INSTALL_LTDL
 else
   LIBLTDL=
 endif # WITH_LTDL
--- ./Makefile.am.org	2005-09-28 17:04:01.000000000 +0000
+++ ./Makefile.am	2005-09-28 17:05:13.000000000 +0000
@@ -113,7 +113,6 @@
 	$(WAND_EXTRA_DIST)
 
 lib_LTLIBRARIES = $(LIBMAGICK) $(LIBWAND) $(LIBMAGICKPP)
-noinst_LTLIBRARIES = $(LIBLTDL)
 
 check_PROGRAMS = \
 	$(TESTS_CHECK_PGRMS) \
--- ./configure.ac.org	2005-09-28 17:05:32.000000000 +0000
+++ ./configure.ac	2005-09-28 17:06:18.000000000 +0000
@@ -293,7 +293,7 @@
 
 # Configure libltdl
 #AC_CONFIG_SUBDIRS(ltdl)
-AC_LIBLTDL_CONVENIENCE(ltdl)
+AC_LIBLTDL_INSTALLABLE(ltdl)
 AC_LIB_LTDL
 # Substitute INCLTDL and LIBLTDL in the Makefiles
 #AC_SUBST(INCLTDL)
