--- ImageMagick-6.6.5-2/configure.ac.orig	2010-10-23 16:19:33.019743558 +0200
+++ ImageMagick-6.6.5-2/configure.ac	2010-10-23 16:51:41.847742720 +0200
@@ -385,10 +385,6 @@
             AC_MSG_WARN([Replacing compiler $CC with compiler $PTHREAD_CC to support pthreads.])
             CC="$PTHREAD_CC"
         fi
-        if test "$CXX" != "$PTHREAD_CXX"; then
-          AC_MSG_WARN([Replacing compiler $CXX with compiler $PTHREAD_CXX to support pthreads.])
-          CXX="$PTHREAD_CXX"
-        fi
         AC_DEFINE(THREAD_SUPPORT,1,[Define if you have POSIX threads libraries and header files.])
     fi
 fi
