diff -urNp -x '*.orig' ImageMagick-6.9.11-34.org/configure.ac ImageMagick-6.9.11-34/configure.ac
--- ImageMagick-6.9.11-34.org/configure.ac	2021-05-07 22:46:18.273024527 +0200
+++ ImageMagick-6.9.11-34/configure.ac	2021-05-07 22:46:19.003023227 +0200
@@ -3891,7 +3891,23 @@ MAGICK_CFLAGS=$CFLAGS
 MAGICK_CXXFLAGS="$CXXFLAGS"
 MAGICK_CPPFLAGS=`echo $MAGICK_CPPFLAGS | sed -e 's/  */ /g'`
 MAGICK_PCFLAGS=`echo $MAGICK_PCFLAGS | sed -e 's/  */ /g'`
-MAGICK_LDFLAGS="-L$LIB_DIR $LDFLAGS"
+MAGICK_LDFLAGS=""
+new_LDFLAGS=
+for flag in $LDFLAGS
+do
+  case "$flag" in
+    -L/usr/lib|-L/usr/lib64|-L/usr/libx32)
+      ;;
+    -s|-Wl,-s|--strip-all|-Wl,--strip-all|-Wl,--as-needed|-Wl,--no-copy-dt-needed-entries)
+      new_LDFLAGS="$new_LDFLAGS $flag"
+      ;;
+    *)
+      MAGICK_LDFLAGS="$MAGICK_LDFLAGS $flag"
+      new_LDFLAGS="$new_LDFLAGS $flag"
+      ;;
+  esac
+done
+LDFLAGS="$new_LDFLAGS"
 MAGICK_LIBS="$MAGICK_DEP_LIBS $MAGICK_EXTRA_DEP_LIBS"
 
 AC_SUBST([MAGICK_CFLAGS])
