--- ImageMagick-6.9.4-6.orig/configure.ac	2016-06-01 00:23:05.000000000 +0200
+++ ImageMagick-6.9.4-6/configure.ac	2016-06-05 16:06:31.718121600 +0200
@@ -703,7 +703,7 @@
     [enable_indirect_reads=$enableval],
     [enable_indirect_reads='no'])
 
-if test "$enable_indirect-reads" = 'yes'; then
+if test "$enable_indirect_reads" = 'yes'; then
     AC_DEFINE(INDIRECT_READS_SUPPORT,1,[enable indirect reads (@) in filenames])
     MAGICK_FEATURES="IndirectReads $MAGICK_FEATURES"
 fi
