--- PerlMagick/demo.paths/button.pl	Mon Nov 23 23:49:13 1998
+++ PerlMagick/demo/button.pl	Sat Feb 19 21:25:55 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # Make simple beveled button.
 #
--- PerlMagick/demo.paths/demo.pl	Mon Jan 11 21:02:02 1999
+++ PerlMagick/demo/demo.pl	Sat Feb 19 21:25:55 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # Overall demo of the major PerlMagick methods.
 #
--- PerlMagick/demo.paths/lsys.pl	Mon Nov 23 23:49:13 1998
+++ PerlMagick/demo/lsys.pl	Sat Feb 19 21:25:55 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # Written by jreed@itis.com, adapted by John Cristy.
 
--- PerlMagick/demo.paths/piddle.pl	Sat May 15 17:48:18 1999
+++ PerlMagick/demo/piddle.pl	Sat Feb 19 21:25:55 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 # Piddle example using PerlMagick methods.
 
 use Image::Magick;
--- PerlMagick/demo.paths/shadow_text.pl	Mon Nov 23 23:49:13 1998
+++ PerlMagick/demo/shadow_text.pl	Sat Feb 19 21:25:55 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # Make simple text with a shadow.
 #
--- PerlMagick/demo.paths/shapes.pl	Fri Jan 22 22:24:39 1999
+++ PerlMagick/demo/shapes.pl	Sat Feb 19 21:25:55 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 # GD example using PerlMagick methods.
 
 use Image::Magick;
--- PerlMagick/demo.paths/steganography.pl	Mon Nov 23 23:49:13 1998
+++ PerlMagick/demo/steganography.pl	Sat Feb 19 21:25:55 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use Image::Magick;
 
--- PerlMagick/demo.paths/tree.pl	Mon Nov 23 23:49:13 1998
+++ PerlMagick/demo/tree.pl	Sat Feb 19 21:25:56 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 require "lsys.pl";
 
 %rule = (
