commit 51968547b98bde1cd6c6cc7145faf3cc1f365a22
Author: Cristy <mikayla-grace@urban-warrior.org>
Date:   Sat Feb 22 09:54:09 2020 -0500

    https://github.com/ImageMagick/ImageMagick/issues/1848

diff --git a/coders/svg.c b/coders/svg.c
index 199475a49..172ec558c 100644
--- a/coders/svg.c
+++ b/coders/svg.c
@@ -3931,7 +3931,7 @@ static MagickBooleanType TraceSVGImage(Image *image,ExceptionInfo *exception)
 {
 #if defined(MAGICKCORE_AUTOTRACE_DELEGATE)
   {
-    at_bitmap_type
+    at_bitmap
       *trace;
 
     at_fitting_opts_type
diff --git a/configure.ac b/configure.ac
index cc5d2344c..9eec1941a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1714,7 +1714,7 @@ AUTOTRACE_LIBS=""
 AUTOTRACE_PKG=""
 if test "x$with_autotrace" = "xyes"; then
   AC_MSG_RESULT([-------------------------------------------------------------])
-  PKG_CHECK_MODULES([AUTOTRACE],[autotrace >= 0.31.1],[have_autotrace=yes],[have_autotrace=no])
+  PKG_CHECK_MODULES([AUTOTRACE],[autotrace >= 0.31.2],[have_autotrace=yes],[have_autotrace=no])
   AC_MSG_RESULT([])
 fi
 
