--- ImageMagick-6.9.10-19/configure.ac.orig	2018-12-21 07:12:50.451777763 +0100
+++ ImageMagick-6.9.10-19/configure.ac	2018-12-21 07:16:16.398442079 +0100
@@ -488,10 +488,6 @@
         CFLAGS="$CFLAGS $DEF_THREAD"
         CXXFLAGS="$CXXFLAGS $DEF_THREAD"
         THREAD_LIBS="$PTHREAD_LIBS"
-        if test "$CC" != "$PTHREAD_CC"; then
-            AC_MSG_WARN([Replacing compiler $CC with compiler $PTHREAD_CC to support pthreads.])
-            CC="$PTHREAD_CC"
-        fi
         AC_DEFINE(THREAD_SUPPORT,1,[Define if you have POSIX threads libraries and header files.])
     fi
 fi
