--- ImageMagick-6.3.2/configure.ac~	2007-02-25 20:25:43.000000000 +0100
+++ ImageMagick-6.3.2/configure.ac	2007-02-25 20:28:04.000000000 +0100
@@ -2763,7 +2763,17 @@
 MAGICK_CXXFLAGS=$CXXFLAGS
 MAGICK_CPPFLAGS=`echo $MAGICK_CPPFLAGS | sed -e 's/  */ /g'`
 MAGICK_PCFLAGS=`echo $MAGICK_PCFLAGS | sed -e 's/  */ /g'`
-MAGICK_LDFLAGS="-L$LIB_DIR $LDFLAGS"
+MAGICK_LDFLAGS=-L$LIB_DIR
+for flag in $LDFLAGS
+do
+  case "$flag" in
+    -s|-Wl,-s|--strip-all|-Wl,--strip-all|-Wl,--as-needed)
+      ;;
+    *)
+      MAGICK_LDFLAGS="$MAGICK_LDFLAGS $flag"
+      ;;
+  esac
+done
 MAGICK_LIBS="-lMagick $MAGICK_DEP_LIBS"
 
 AC_SUBST(MAGICK_CFLAGS)
