--- ImageMagick-5.4.7/Makefile.am~	Wed Jul  3 21:14:23 2002
+++ ImageMagick-5.4.7/Makefile.am	Wed Jul  3 21:14:23 2002
@@ -28,7 +28,7 @@
 # ImageMagick Studio.
 
 # Don't require all the GNU mandated files
-AUTOMAKE_OPTIONS = 1.6.2 dist-zip dist-bzip2 foreign
+AUTOMAKE_OPTIONS = dist-zip dist-bzip2 foreign
 
 # Subdirectories to build
 if WITH_MAGICK_PLUS_PLUS
--- ImageMagick-5.4.7/ltdl/Makefile.am~	Wed Jul  3 21:14:23 2002
+++ ImageMagick-5.4.7/ltdl/Makefile.am	Wed Jul  3 21:14:23 2002
@@ -27,7 +27,7 @@
 # ImageMagick Studio.
 #
 
-AUTOMAKE_OPTIONS = 1.6.2 foreign
+AUTOMAKE_OPTIONS = foreign
 
 if WITH_MODULES
 
