--- ImageMagick-6.9.10-19/configure.ac.orig	2018-12-21 07:11:13.258445539 +0100
+++ ImageMagick-6.9.10-19/configure.ac	2018-12-21 07:12:20.068444776 +0100
@@ -3937,7 +3937,23 @@
 MAGICK_CXXFLAGS="$CXXFLAGS"
 MAGICK_CPPFLAGS=`echo $MAGICK_CPPFLAGS | sed -e 's/  */ /g'`
 MAGICK_PCFLAGS=`echo $MAGICK_PCFLAGS | sed -e 's/  */ /g'`
-MAGICK_LDFLAGS="-L$LIB_DIR $LDFLAGS"
+MAGICK_LDFLAGS=""
+new_LDFLAGS=
+for flag in $LDFLAGS
+do
+  case "$flag" in
+    -L/usr/lib|-L/usr/lib64|-L/usr/libx32)
+      ;;
+    -s|-Wl,-s|--strip-all|-Wl,--strip-all|-Wl,--as-needed|-Wl,--no-copy-dt-needed-entries)
+      new_LDFLAGS="$new_LDFLAGS $flag"
+      ;;
+    *)
+      MAGICK_LDFLAGS="$MAGICK_LDFLAGS $flag"
+      new_LDFLAGS="$new_LDFLAGS $flag"
+      ;;
+  esac
+done
+LDFLAGS="$new_LDFLAGS"
 MAGICK_LIBS="$MAGICK_DEP_LIBS $MAGICK_EXTRA_DEP_LIBS"
 
 AC_SUBST(MAGICK_CFLAGS)
