--- ImageMagick-6.9.11-34/configure.ac.orig	2020-10-31 22:00:55.255994873 +0100
+++ ImageMagick-6.9.11-34/configure.ac	2020-10-31 22:03:29.635158530 +0100
@@ -489,10 +489,6 @@
         CFLAGS="$CFLAGS $DEF_THREAD"
         CXXFLAGS="$CXXFLAGS $DEF_THREAD"
         THREAD_LIBS="$PTHREAD_LIBS"
-        if test "$CC" != "$PTHREAD_CC"; then
-            AC_MSG_WARN([Replacing compiler $CC with compiler $PTHREAD_CC to support pthreads.])
-            CC="$PTHREAD_CC"
-        fi
         AC_DEFINE([THREAD_SUPPORT],[1],[Define if you have POSIX threads libraries and header files.])
     fi
 fi
