--- ImageMagick-7.0.10-35/configure.ac.orig	2020-11-01 08:45:30.469806589 +0100
+++ ImageMagick-7.0.10-35/configure.ac	2020-11-01 09:05:17.360043320 +0100
@@ -489,10 +489,6 @@
         CFLAGS="$CFLAGS $DEF_THREAD"
         CXXFLAGS="$CXXFLAGS $DEF_THREAD"
         THREAD_LIBS="$PTHREAD_LIBS"
-        if test "$CC" != "$PTHREAD_CC"; then
-            AC_MSG_WARN([Replacing compiler $CC with compiler $PTHREAD_CC to support pthreads.])
-            CC="$PTHREAD_CC"
-        fi
         AC_DEFINE([THREAD_SUPPORT],[1],[Define if you have POSIX threads libraries and header files.])
     fi
 fi
