--- ImageMagick-6.4.1/configure.ac.orig	2008-05-27 09:44:58.000000000 +0200
+++ ImageMagick-6.4.1/configure.ac	2008-05-27 09:46:59.000000000 +0200
@@ -3086,7 +3086,17 @@
 MAGICK_CXXFLAGS="$CXXFLAGS"
 MAGICK_CPPFLAGS=`echo $MAGICK_CPPFLAGS | sed -e 's/  */ /g'`
 MAGICK_PCFLAGS=`echo $MAGICK_PCFLAGS | sed -e 's/  */ /g'`
-MAGICK_LDFLAGS="-L$LIB_DIR $LDFLAGS"
+MAGICK_LDFLAGS="-L$LIB_DIR"
+for flag in $LDFLAGS
+do
+  case "$flag" in
+    -s|-Wl,-s|--strip-all|-Wl,--strip-all|-Wl,--as-needed)
+      ;;
+    *)
+      MAGICK_LDFLAGS="$MAGICK_LDFLAGS $flag"
+      ;;
+  esac
+done
 MAGICK_LIBS="-lMagickCore $MAGICK_DEP_LIBS $MAGICK_API_LIBLTDL"
 
 AC_SUBST(MAGICK_CFLAGS)
