--- ImageMagick-5.3.6/Makefile.am~	Thu Jun 14 17:54:36 2001
+++ ImageMagick-5.3.6/Makefile.am	Thu Jun 28 13:05:13 2001
@@ -77,18 +77,18 @@
 pkgdocdir = @MagickSharePath@
 DOCDIRS = images www www/api www/api/types www/Magick++
 install-data-html:
-	$(mkinstalldirs) $(pkgdocdir)
-	$(INSTALL_DATA) $(srcdir)/ImageMagick.html $(pkgdocdir)
+	$(mkinstalldirs) $(DESTDIR)$(pkgdocdir)
+	$(INSTALL_DATA) $(srcdir)/ImageMagick.html $(DESTDIR)$(pkgdocdir)
 	for dir in $(DOCDIRS) ; do \
-	  $(mkinstalldirs) $(pkgdocdir)/$$dir && \
-	  $(INSTALL_DATA) $(srcdir)/$$dir/*.* $(pkgdocdir)/$$dir ; \
+	  $(mkinstalldirs) $(DESTDIR)$(pkgdocdir)/$$dir && \
+	  $(INSTALL_DATA) $(srcdir)/$$dir/*.* $(DESTDIR)$(pkgdocdir)/$$dir ; \
 	done
 
 # Uninstall HTML files
 uninstall-data-html:
-	rm -f $(pkgdocdir)/ImageMagick.html
+	rm -f $(DESTDIR)$(pkgdocdir)/ImageMagick.html
 	for dir in $(DOCDIRS) ; do \
-	  rm -rf $(pkgdocdir)/$$dir ; \
+	  rm -rf $(DESTDIR)$(pkgdocdir)/$$dir ; \
 	done
 
 if WITH_PERL
