--- ImageMagick-5.5.2/configure.ac.wiget	Wed Dec 18 01:50:24 2002
+++ ImageMagick-5.5.2/configure.ac	Wed Dec 18 01:51:19 2002
@@ -943,8 +943,8 @@
     O_CPPFLAGS="$CPPFLAGS"
     CPPFLAGS="$CPPFLAGS -I${ac_x_includes}/X11"
     AC_CHECK_HEADER(DPS/dpsXclient.h,passed=`expr $passed + 1`,failed=`expr $failed + 1`)
-    AC_CHECK_LIB(dps,DPSInitialize,passed=`expr $passed + 1`,failed=`expr $failed + 1`,)
-    AC_CHECK_LIB(dpstk,XDPSPixelsPerPoint,passed=`expr $passed + 1`,failed=`expr $failed + 1`,-ldps)
+    AC_CHECK_LIB(dps,DPSInitialize,passed=`expr $passed + 1`,failed=`expr $failed + 1`,-lXt)
+    AC_CHECK_LIB(dpstk,XDPSPixelsPerPoint,passed=`expr $passed + 1`,failed=`expr $failed + 1`,-ldps -lXt)
     AC_MSG_CHECKING(if DPS package is complete)
     if test $passed -gt 0
     then
@@ -954,7 +954,7 @@
         have_dps='no (failed tests)'
 	CPPFLAGS="$O_CPPFLAGS"
     else
-	LIB_DPS='-ldpstk -ldps'
+	LIB_DPS='-ldpstk -ldps -lXt'
 	LIBS="$LIB_DPS $LIBS"
 	AC_DEFINE(HasDPS,,Define if you have Display Postscript)
 	AC_MSG_RESULT(yes)
