--- ImageMagick-7.1.1-31/MagickCore/opencl.c.orig	2024-04-25 20:19:45.713297893 +0200
+++ ImageMagick-7.1.1-31/MagickCore/opencl.c	2024-04-25 20:53:35.772300120 +0200
@@ -2484,7 +2484,7 @@ static MagickBooleanType BindOpenCLFunct
 #ifdef MAGICKCORE_WINDOWS_SUPPORT
   openCL_library->library=(void *)lt_dlopen("OpenCL.dll");
 #else
-  openCL_library->library=(void *)lt_dlopen("libOpenCL.so");
+  openCL_library->library=(void *)lt_dlopen("libOpenCL.so.1");
 #endif
 #define BIND(X) \
   if ((openCL_library->X=(MAGICKpfn_##X)OsLibraryGetFunctionAddress(openCL_library->library,#X)) == NULL) \
