--- ImageMagick-6.8.4-8/configure.ac.orig	2013-04-09 15:47:16.713625903 +0200
+++ ImageMagick-6.8.4-8/configure.ac	2013-04-09 15:49:29.416956473 +0200
@@ -3903,7 +3903,17 @@
 MAGICK_CXXFLAGS="$CXXFLAGS"
 MAGICK_CPPFLAGS=`echo $MAGICK_CPPFLAGS | sed -e 's/  */ /g'`
 MAGICK_PCFLAGS=`echo $MAGICK_PCFLAGS | sed -e 's/  */ /g'`
-MAGICK_LDFLAGS="-L$LIB_DIR $LDFLAGS"
+MAGICK_LDFLAGS=""
+for flag in $LDFLAGS
+do
+  case "$flag" in
+    -s|-Wl,-s|--strip-all|-Wl,--strip-all|-Wl,--as-needed)
+      ;;
+    *)
+      MAGICK_LDFLAGS="$MAGICK_LDFLAGS $flag"
+      ;;
+  esac
+done
 MAGICK_LIBS="$MAGICK_DEP_LIBS"
 
 AC_SUBST(MAGICK_CFLAGS)
