--- ImageMagick-7.0.8-14/configure.ac~	2018-10-24 20:04:34.000000000 +0300
+++ ImageMagick-7.0.8-14/configure.ac	2018-10-24 20:06:02.215578717 +0300
@@ -3939,7 +3939,22 @@
 MAGICK_CXXFLAGS="$CXXFLAGS"
 MAGICK_CPPFLAGS=`echo $MAGICK_CPPFLAGS | sed -e 's/  */ /g'`
 MAGICK_PCFLAGS=`echo $MAGICK_PCFLAGS | sed -e 's/  */ /g'`
-MAGICK_LDFLAGS="-L$LIB_DIR $LDFLAGS"
+MAGICK_LDFLAGS=''
+new_LDFLAGS=''
+for flag in $LDFLAGS; do
+  case "$flag" in
+    -L/usr/lib|-L/usr/lib64|-L/usr/libx32)
+      ;;
+    -s|-Wl,-s|--strip-all|-Wl,--strip-all|-Wl,--as-needed|-Wl,--no-copy-dt-needed-entries)
+      new_LDFLAGS="$new_LDFLAGS $flag"
+      ;;
+    *)
+      MAGICK_LDFLAGS="$MAGICK_LDFLAGS $flag"
+      new_LDFLAGS="$new_LDFLAGS $flag"
+      ;;
+  esac
+done
+LDFLAGS="$new_LDFLAGS"
 MAGICK_LIBS="$MAGICK_DEP_LIBS MAGICK_EXTRA_DEP_LIBS"
 
 AC_SUBST(MAGICK_CFLAGS)
