--- ImageMagick-6.8.7-3/configure.ac.orig	2013-11-01 09:48:55.513688836 +0100
+++ ImageMagick-6.8.7-3/configure.ac	2013-11-02 15:48:26.215824407 +0100
@@ -2515,14 +2515,14 @@
     failed=0
     passed=0
     AC_CHECK_HEADER(mupdf/fitz.h,passed=`expr $passed + 1`,failed=`expr $failed + 1`,)
-    AC_CHECK_LIB(mupdf,fz_new_context,passed=`expr $passed + 1`,failed=`expr $failed + 1`,-lmupdf $FREETYPE_LIBS)
+    AC_CHECK_LIB(mupdf,fz_new_context,passed=`expr $passed + 1`,failed=`expr $failed + 1`,-lmupdf-js-none $FREETYPE_LIBS)
     AC_MSG_CHECKING([if MUPDF package is complete])
     if test $passed -gt 0; then
         if test $failed -gt 0; then
             AC_MSG_RESULT([no -- some components failed test])
             have_mupdf='no (failed tests)'
         else
-            MUPDF_LIBS='-lmupdf'
+            MUPDF_LIBS='-lmupdf -lmupdf-js-none'
             LIBS="$MUPDF_LIBS $LIBS"
             AC_DEFINE(MUPDF_DELEGATE,1,Define if you have MUPDF library)
             AC_MSG_RESULT([yes])
