--- ImageMagick-6.4.1/configure.ac.orig	2008-05-11 07:11:08.000000000 +0200
+++ ImageMagick-6.4.1/configure.ac	2008-05-27 09:28:15.000000000 +0200
@@ -2191,9 +2192,6 @@
         if test -d "${xml2_prefix}/include/libxml2"; then
             CPPFLAGS="$CPPFLAGS -I${xml2_prefix}/include/libxml2"
         fi
-        if test "${xml2_prefix}" != '/usr'; then
-            LDFLAGS="$LDFLAGS -L${xml2_prefix}/lib"
-        fi
     fi
     failed=0
     passed=0
