--- ImageMagick-5.5.1/configure.ac.orig	Fri Nov  1 14:37:40 2002
+++ ImageMagick-5.5.1/configure.ac	Fri Nov  1 14:41:30 2002
@@ -148,7 +148,7 @@
 
 # Configure libltdl
 #AC_CONFIG_SUBDIRS(ltdl)
-AC_LIBLTDL_CONVENIENCE(ltdl)
+AC_LIBLTDL_INSTALLABLE(ltdl)
 AC_LIB_LTDL
 # Substitute INCLTDL and LIBLTDL in the Makefiles
 AC_SUBST(INCLTDL)
--- ImageMagick-5.5.1/utilities/Makefile.am.orig	Fri Oct 11 22:07:21 2002
+++ ImageMagick-5.5.1/utilities/Makefile.am	Fri Nov  1 15:11:18 2002
@@ -58,7 +58,7 @@
 
 # Libtool libraries, dependencies, and linker flags used by most executables
 if WITH_MODULES
-COMMONLDADD	= $(MAGICK_LIB) $(top_builddir)/ltdl/libltdlc.la
+COMMONLDADD	= $(MAGICK_LIB)
 COMMONLDFLAGS	= @LDFLAGS@ @DLLDFLAGS@
 else
 COMMONLDADD	= $(MAGICK_LIB)
