--- ImageMagick-5.5.7/configure.ac.orig	2003-05-31 16:41:34.000000000 +0200
+++ ImageMagick-5.5.7/configure.ac	2003-07-18 08:27:51.000000000 +0200
@@ -2194,7 +2194,7 @@
 # Determine path to pick up ImageMagick library from for use with building PerlMagick
 MAGICKLIBDIR="${LIB_DIR}"
 MAGICKLIB="-L${MAGICKLIBDIR} -lMagick"
-if test $with_perl_static = 'yes'
+if test $with_perl_static = 'yes' -o $with_perl_dynamic = 'yes'
 then
   # Find out where libtool hides its uninstalled libraries (as libtool_objdir)
   eval `./libtool --config|grep '^objdir='|sed -e 's/^objdir/libtool_objdir/'`
--- ImageMagick-5.5.7/Makefile.am.orig	2003-05-15 17:24:39.000000000 +0200
+++ ImageMagick-5.5.7/Makefile.am	2003-07-18 08:27:25.000000000 +0200
@@ -97,11 +97,12 @@
 	cd $(PERLMAGICK) && @PERL@ Makefile.PL $(PERL_MAKE_OPTIONS)
 
 install-exec-perl: $(PERLMAKEFILE)
-	( cd $(PERLMAGICK) && $(MAKE) CC='@CC@' && \
+	( cd $(PERLMAGICK) && \
 	$(MAKE) CC='@CC@' install && \
 	$(MAKE) clean && rm -f  Makefile.old )
 
-all-perl:
+all-perl: $(PERLMAKEFILE) magick/libMagick.la
+	cd $(PERLMAGICK) && $(MAKE) CC='@CC@'
 
 uninstall-exec-perl: $(PERLMAKEFILE)
 	echo "Uninstall not supported for PerlMagick"
