--- ImageMagick-6.6.6-7/configure.ac~	2010-12-27 13:27:17.432375100 +0100
+++ ImageMagick-6.6.6-7/configure.ac	2010-12-27 13:30:28.809041767 +0100
@@ -381,10 +381,6 @@
         DEF_THREAD="$PTHREAD_CFLAGS"
         CFLAGS="$CFLAGS $DEF_THREAD"
         CXXFLAGS="$CXXFLAGS $DEF_THREAD"
-        if test "$CC" != "$PTHREAD_CC"; then
-            AC_MSG_WARN([Replacing compiler $CC with compiler $PTHREAD_CC to support pthreads.])
-            CC="$PTHREAD_CC"
-        fi
         AC_DEFINE(THREAD_SUPPORT,1,[Define if you have POSIX threads libraries and header files.])
     fi
 fi
