--- ImageMagick-7.1.2-9/coders/dng.c.orig	2025-11-30 14:26:20.000000000 +0100
+++ ImageMagick-7.1.2-9/coders/dng.c	2026-01-13 21:06:57.346461706 +0100
@@ -332,7 +332,7 @@
 }
 
 static void LibRawDataError(void *data,const char *magick_unused(file),
-#if defined(MAGICK_LIBRAW_VERSION_TAIL) && MAGICK_LIBRAW_VERSION_TAIL == 202502
+#if defined(LIBRAW_MAJOR_VERSION) && defined(LIBRAW_MINOR_VERSION) && (LIBRAW_MAJOR_VERSION > 0 || LIBRAW_MINOR_VERSION >= 22)
   const INT64 offset)
 #else
   const int offset)
