diff -urNp -x '*.orig' ImageMagick-6.9.11-34.org/magick/Magick-config.in ImageMagick-6.9.11-34/magick/Magick-config.in
--- ImageMagick-6.9.11-34.org/magick/Magick-config.in	2020-10-08 15:25:41.000000000 +0200
+++ ImageMagick-6.9.11-34/magick/Magick-config.in	2021-05-07 22:46:23.866347708 +0200
@@ -8,11 +8,11 @@ prefix=@prefix@
 exec_prefix=@exec_prefix@
 
 usage="\
-Usage: Magick-config [--cflags] [--cppflags] [--exec-prefix] [--ldflags] [--libs] [--prefix] [--version]"
+Usage: Magick6-config [--cflags] [--cppflags] [--exec-prefix] [--ldflags] [--libs] [--prefix] [--version]"
 
 if test $# -eq 0; then
       echo "${usage}" 1>&2
-      echo "Example: gcc \`Magick-config --cflags --cppflags\` -o core core.c \`Magick-config --ldflags --libs\`" 1>&2
+      echo "Example: gcc \`Magick6-config --cflags --cppflags\` -o core core.c \`Magick6-config --ldflags --libs\`" 1>&2
       exit 1
 fi
 
@@ -38,19 +38,19 @@ while test $# -gt 0; do
       echo '@PACKAGE_VERSION@ Q@QUANTUM_DEPTH@ @MAGICK_HDRI@'
       ;;
     --cflags)
-      @PKG_CONFIG@ --cflags MagickCore
+      @PKG_CONFIG@ --cflags MagickCore6
       ;;
     --cxxflags)
-      @PKG_CONFIG@ --cflags MagickCore
+      @PKG_CONFIG@ --cflags MagickCore6
       ;;
     --cppflags)
-      @PKG_CONFIG@ --cflags MagickCore
+      @PKG_CONFIG@ --cflags MagickCore6
       ;;
     --ldflags)
-      @PKG_CONFIG@ --libs MagickCore
+      @PKG_CONFIG@ --libs MagickCore6
       ;;
     --libs)
-      @PKG_CONFIG@ --libs MagickCore
+      @PKG_CONFIG@ --libs MagickCore6
       ;;
     *)
       echo "${usage}" 1>&2
diff -urNp -x '*.orig' ImageMagick-6.9.11-34.org/magick/MagickCore-config.in ImageMagick-6.9.11-34/magick/MagickCore-config.in
--- ImageMagick-6.9.11-34.org/magick/MagickCore-config.in	2020-10-08 15:25:41.000000000 +0200
+++ ImageMagick-6.9.11-34/magick/MagickCore-config.in	2021-05-07 22:46:23.866347708 +0200
@@ -8,11 +8,11 @@ prefix=@prefix@
 exec_prefix=@exec_prefix@
 
 usage="\
-Usage: MagickCore-config [--cflags] [--cppflags] [--exec-prefix] [--ldflags] [--libs] [--prefix] [--version]"
+Usage: MagickCore6-config [--cflags] [--cppflags] [--exec-prefix] [--ldflags] [--libs] [--prefix] [--version]"
 
 if test $# -eq 0; then
       echo "${usage}" 1>&2
-      echo "Example: gcc \`MagickCore-config --cflags --cppflags\` -o core core.c \`Magick-config --ldflags --libs\`" 1>&2
+      echo "Example: gcc \`MagickCore6-config --cflags --cppflags\` -o core core.c \`Magick6-config --ldflags --libs\`" 1>&2
       exit 1
 fi
 
diff -urNp -x '*.orig' ImageMagick-6.9.11-34.org/wand/MagickWand-config.in ImageMagick-6.9.11-34/wand/MagickWand-config.in
--- ImageMagick-6.9.11-34.org/wand/MagickWand-config.in	2020-10-08 15:25:41.000000000 +0200
+++ ImageMagick-6.9.11-34/wand/MagickWand-config.in	2021-05-07 22:46:23.866347708 +0200
@@ -8,11 +8,11 @@ prefix=@prefix@
 exec_prefix=@exec_prefix@
 
 usage="\
-Usage: MagickWand-config [--cflags] [--cppflags] [--exec-prefix] [--ldflags] [--libs] [--prefix] [--version]"
+Usage: MagickWand6-config [--cflags] [--cppflags] [--exec-prefix] [--ldflags] [--libs] [--prefix] [--version]"
 
 if test $# -eq 0; then
       echo "${usage}" 1>&2
-      echo "Example: gcc \`MagickWand-config --cflags --cppflags\` -o wand wand.c \`MagickWand-config --ldflags --libs\`" 1>&2
+      echo "Example: gcc \`MagickWand6-config --cflags --cppflags\` -o wand wand.c \`MagickWand6-config --ldflags --libs\`" 1>&2
       exit 1
 fi
 
diff -urNp -x '*.orig' ImageMagick-6.9.11-34.org/wand/Wand-config.in ImageMagick-6.9.11-34/wand/Wand-config.in
--- ImageMagick-6.9.11-34.org/wand/Wand-config.in	2020-10-08 15:25:41.000000000 +0200
+++ ImageMagick-6.9.11-34/wand/Wand-config.in	2021-05-07 22:46:23.866347708 +0200
@@ -8,11 +8,11 @@ prefix=@prefix@
 exec_prefix=@exec_prefix@
 
 usage="\
-Usage: Wand-config [--cflags] [--cppflags] [--exec-prefix] [--ldflags] [--libs] [--prefix] [--version]"
+Usage: Wand6-config [--cflags] [--cppflags] [--exec-prefix] [--ldflags] [--libs] [--prefix] [--version]"
 
 if test $# -eq 0; then
       echo "${usage}" 1>&2
-      echo "Example: gcc \`Wand-config --cflags --cppflags\` -o wand wand.c \`Wand-config --ldflags --libs\`" 1>&2
+      echo "Example: gcc \`Wand6-config --cflags --cppflags\` -o wand wand.c \`Wand6-config --ldflags --libs\`" 1>&2
       exit 1
 fi
 
@@ -38,19 +38,19 @@ while test $# -gt 0; do
       echo '@PACKAGE_VERSION@ Q@QUANTUM_DEPTH@ @MAGICK_HDRI@'
       ;;
     --cflags)
-      @PKG_CONFIG@ --cflags MagickWand
+      @PKG_CONFIG@ --cflags MagickWand6
       ;;
     --cxxflags)
-      @PKG_CONFIG@ --cflags MagickWand
+      @PKG_CONFIG@ --cflags MagickWand6
       ;;
     --cppflags)
-      @PKG_CONFIG@ --cflags MagickWand
+      @PKG_CONFIG@ --cflags MagickWand6
       ;;
     --ldflags)
-      @PKG_CONFIG@ --libs MagickWand
+      @PKG_CONFIG@ --libs MagickWand6
       ;;
     --libs)
-      @PKG_CONFIG@ --libs MagickWand
+      @PKG_CONFIG@ --libs MagickWand6
       ;;
     *)
       echo "${usage}" 1>&2
diff -urNp -x '*.orig' ImageMagick-6.9.11-34.org/wand/Wand.pc.in ImageMagick-6.9.11-34/wand/Wand.pc.in
--- ImageMagick-6.9.11-34.org/wand/Wand.pc.in	2020-10-08 15:25:41.000000000 +0200
+++ ImageMagick-6.9.11-34/wand/Wand.pc.in	2021-05-07 22:46:23.866347708 +0200
@@ -9,7 +9,7 @@ Name: MagickWand
 Description: MagickCore - C API for ImageMagick (ABI @MAGICK_ABI_SUFFIX@)
 URL: https://github.com/ImageMagick/ImageMagick6
 Version: @PACKAGE_VERSION@
-Requires: MagickCore
+Requires: MagickCore-@MAGICK_MAJOR_VERSION@.@MAGICK_ABI_SUFFIX@
 Cflags: -I${includearchdir} -I${includedir} @MAGICK_PCFLAGS@
 Libs: -L${libdir} -l${libname}
 Libs.private: -L${libdir} -l${libname} @MAGICK_LIBS@ @MATH_LIBS@
