--- ImageMagick-6.9.9-26/magick/Magick-config.in~	2017-12-14 19:42:48.000000000 +0200
+++ ImageMagick-6.9.9-26/magick/Magick-config.in	2017-12-14 19:51:25.404235766 +0200
@@ -8,11 +8,11 @@
 exec_prefix=@exec_prefix@
 
 usage="\
-Usage: Magick-config [--cflags] [--cppflags] [--exec-prefix] [--ldflags] [--libs] [--prefix] [--version]"
+Usage: Magick6-config [--cflags] [--cppflags] [--exec-prefix] [--ldflags] [--libs] [--prefix] [--version]"
 
 if test $# -eq 0; then
       echo "${usage}" 1>&2
-      echo "Example: gcc \`Magick-config --cflags --cppflags\` -o core core.c \`Magick-config --ldflags --libs\`" 1>&2
+      echo "Example: gcc \`Magick6-config --cflags --cppflags\` -o core core.c \`Magick6-config --ldflags --libs\`" 1>&2
       exit 1
 fi
 
@@ -38,19 +38,19 @@
       echo '@PACKAGE_VERSION@ Q@QUANTUM_DEPTH@ @MAGICK_HDRI@'
       ;;
     --cflags)
-      @PKG_CONFIG@ --cflags MagickCore
+      @PKG_CONFIG@ --cflags MagickCore6
       ;;
     --cxxflags)
-      @PKG_CONFIG@ --cflags MagickCore
+      @PKG_CONFIG@ --cflags MagickCore6
       ;;
     --cppflags)
-      @PKG_CONFIG@ --cflags MagickCore
+      @PKG_CONFIG@ --cflags MagickCore6
       ;;
     --ldflags)
-      @PKG_CONFIG@ --libs MagickCore
+      @PKG_CONFIG@ --libs MagickCore6
       ;;
     --libs)
-      @PKG_CONFIG@ --libs MagickCore
+      @PKG_CONFIG@ --libs MagickCore6
       ;;
     *)
       echo "${usage}" 1>&2
--- ImageMagick-6.9.9-26/magick/MagickCore-config.in~	2017-12-14 19:53:01.000000000 +0200
+++ ImageMagick-6.9.9-26/magick/MagickCore-config.in	2017-12-14 19:53:39.770926527 +0200
@@ -8,11 +8,11 @@
 exec_prefix=@exec_prefix@
 
 usage="\
-Usage: MagickCore-config [--cflags] [--cppflags] [--exec-prefix] [--ldflags] [--libs] [--prefix] [--version]"
+Usage: MagickCore6-config [--cflags] [--cppflags] [--exec-prefix] [--ldflags] [--libs] [--prefix] [--version]"
 
 if test $# -eq 0; then
       echo "${usage}" 1>&2
-      echo "Example: gcc \`MagickCore-config --cflags --cppflags\` -o core core.c \`Magick-config --ldflags --libs\`" 1>&2
+      echo "Example: gcc \`MagickCore6-config --cflags --cppflags\` -o core core.c \`Magick6-config --ldflags --libs\`" 1>&2
       exit 1
 fi
 
@@ -38,19 +38,19 @@
       echo '@PACKAGE_VERSION@ Q@QUANTUM_DEPTH@ @MAGICK_HDRI@'
       ;;
     --cflags)
-      @PKG_CONFIG@ -config --cflags MagickCore
+      @PKG_CONFIG@ -config --cflags MagickCore6
       ;;
     --cxxflags)
-      @PKG_CONFIG@ --cflags MagickCore
+      @PKG_CONFIG@ --cflags MagickCore6
       ;;
     --cppflags)
-      @PKG_CONFIG@ --cflags MagickCore
+      @PKG_CONFIG@ --cflags MagickCore6
       ;;
     --ldflags)
-      @PKG_CONFIG@ --libs MagickCore
+      @PKG_CONFIG@ --libs MagickCore6
       ;;
     --libs)
-      @PKG_CONFIG@ --libs MagickCore
+      @PKG_CONFIG@ --libs MagickCore6
       ;;
     *)
       echo "${usage}" 1>&2
--- ImageMagick-6.9.8-9/wand/MagickWand-config.in.orig	2017-06-03 02:38:04.000000000 +0200
+++ ImageMagick-6.9.8-9/wand/MagickWand-config.in	2017-06-03 21:50:38.584589096 +0200
@@ -8,11 +8,11 @@
 exec_prefix=@exec_prefix@
 
 usage="\
-Usage: MagickWand-config [--cflags] [--cppflags] [--exec-prefix] [--ldflags] [--libs] [--prefix] [--version]"
+Usage: MagickWand6-config [--cflags] [--cppflags] [--exec-prefix] [--ldflags] [--libs] [--prefix] [--version]"
 
 if test $# -eq 0; then
       echo "${usage}" 1>&2
-      echo "Example: gcc \`MagickWand-config --cflags --cppflags\` -o wand wand.c \`MagickWand-config --ldflags --libs\`" 1>&2
+      echo "Example: gcc \`MagickWand6-config --cflags --cppflags\` -o wand wand.c \`MagickWand6-config --ldflags --libs\`" 1>&2
       exit 1
 fi
 
@@ -38,19 +38,19 @@
       echo '@PACKAGE_VERSION@ Q@QUANTUM_DEPTH@ @MAGICK_HDRI@'
       ;;
     --cflags)
-      @PKG_CONFIG@ --cflags MagickWand
+      @PKG_CONFIG@ --cflags MagickWand6
       ;;
     --cxxflags)
-      @PKG_CONFIG@ --cflags MagickWand
+      @PKG_CONFIG@ --cflags MagickWand6
       ;;
     --cppflags)
-      @PKG_CONFIG@ --cflags MagickWand
+      @PKG_CONFIG@ --cflags MagickWand6
       ;;
     --ldflags)
-      @PKG_CONFIG@ --libs MagickWand
+      @PKG_CONFIG@ --libs MagickWand6
       ;;
     --libs)
-      @PKG_CONFIG@ --libs MagickWand
+      @PKG_CONFIG@ --libs MagickWand6
       ;;
     *)
       echo "${usage}" 1>&2
--- ImageMagick-6.9.7-0/wand/Wand-config.in~	2016-12-18 14:34:45.000000000 +0100
+++ ImageMagick-6.9.7-0/wand/Wand-config.in	2016-12-27 08:39:14.421666371 +0100
@@ -8,11 +8,11 @@
 exec_prefix=@exec_prefix@
 
 usage="\
-Usage: Wand-config [--cflags] [--cppflags] [--exec-prefix] [--ldflags] [--libs] [--prefix] [--version]"
+Usage: Wand6-config [--cflags] [--cppflags] [--exec-prefix] [--ldflags] [--libs] [--prefix] [--version]"
 
 if test $# -eq 0; then
       echo "${usage}" 1>&2
-      echo "Example: gcc \`Wand-config --cflags --cppflags\` -o wand wand.c \`Wand-config --ldflags --libs\`" 1>&2
+      echo "Example: gcc \`Wand6-config --cflags --cppflags\` -o wand wand.c \`Wand6-config --ldflags --libs\`" 1>&2
       exit 1
 fi
 
@@ -38,19 +38,19 @@
       echo '@PACKAGE_VERSION@ Q@QUANTUM_DEPTH@ @MAGICK_HDRI@'
       ;;
     --cflags)
-      @PKG_CONFIG@ --cflags MagickWand
+      @PKG_CONFIG@ --cflags MagickWand6
       ;;
     --cxxflags)
-      @PKG_CONFIG@ --cflags MagickWand
+      @PKG_CONFIG@ --cflags MagickWand6
       ;;
     --cppflags)
-      @PKG_CONFIG@ --cflags MagickWand
+      @PKG_CONFIG@ --cflags MagickWand6
       ;;
     --ldflags)
-      @PKG_CONFIG@ --libs MagickWand
+      @PKG_CONFIG@ --libs MagickWand6
       ;;
     --libs)
-      @PKG_CONFIG@ --libs MagickWand
+      @PKG_CONFIG@ --libs MagickWand6
       ;;
     *)
       echo "${usage}" 1>&2
--- ImageMagick-6.9.7-0/wand/MagickWand.pc.in~	2016-12-18 14:34:45.000000000 +0100
+++ ImageMagick-6.9.7-0/wand/MagickWand.pc.in	2016-12-27 08:45:54.205764162 +0100
@@ -9,7 +9,7 @@
 Description: MagickWand - C API for ImageMagick (ABI @MAGICK_ABI_SUFFIX@)
 URL: https://www.imagemagick.org
 Version: @PACKAGE_VERSION@
-Requires: MagickCore
+Requires: MagickCore6
 Cflags: -I${includearchdir} -I${includedir} @MAGICK_PCFLAGS@
 Libs: -L${libdir} -l${libname} 
 Libs.private: -L${libdir} -l${libname} @MAGICK_LIBS@ @MATH_LIBS@
--- ImageMagick-6.9.7-0/wand/Wand.pc.in~	2016-12-18 14:34:45.000000000 +0100
+++ ImageMagick-6.9.7-0/wand/Wand.pc.in	2016-12-27 08:46:32.786803064 +0100
@@ -9,7 +9,7 @@
 Description: MagickCore - C API for ImageMagick (ABI @MAGICK_ABI_SUFFIX@)
 URL: https://www.imagemagick.org
 Version: @PACKAGE_VERSION@
-Requires: MagickCore
+Requires: MagickCore6
 Cflags: -I${includearchdir} -I${includedir} @MAGICK_PCFLAGS@
 Libs: -L${libdir} -l${libname}
 Libs.private: -L${libdir} -l${libname} @MAGICK_LIBS@ @MATH_LIBS@
